### DISCUSSION TITLE
graphiql throw Exception : unexpected token {

### DISCUSSION THREAD

**Discussion by PhenJuly** [graphql-java-kickstart/graphql-spring-boot#537] [Q&A]
**Describe the bug**
The browser reported an error that “ unexpected token {  ” while accessing the page /graphiql
(in version 7.0.1 is normal access)

**Desktop (please complete the following information):**
- Browser [ chrome]
- Version [11.0.0]

**Comment by oliemansm** [comment]
@PhenJuly Converted to Discussion since it's working fine for me in latest version. Can you provide more information on steps how to reproduce, because there's not enough information atm to be able to help out.

**Comment by PhenJuly** [comment]
I thought of a possibility that&nbsp;&nbsp;I used the company's intranet, not connected to the Internet,But I'm not sure.I hope this conjecture is helpful to you.


However, interestingly, the version 7.0.1 is used, which can access graphiql normally




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "graphql-java-kickstart/graphql-spring-boot"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月21日(星期天) 晚上9:03
收件人:&nbsp;"graphql-java-kickstart/graphql-spring-boot"<graphql-spring-boot@noreply.github.com&gt;;
抄送:&nbsp;"翎翔羽翼"<527273738@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [graphql-java-kickstart/graphql-spring-boot] graphiql throw Exception : unexpected token { (#537)





 
@PhenJuly Converted to Discussion since it's working fine for me in latest version. Can you provide more information on steps how to reproduce, because there's not enough information atm to be able to help out.
